scripts/ampHtmlValidator/index.js

```diff
@@ -23,6 +23,7 @@ const pageTypes = [
 // list of urls we have decided are acceptable to fail amp validation
 const excludedUrls = [
   '/mundo/23263889' /* https://github.com/bbc/simorgh/issues/8104 */,
+  '/mundo/noticias-internacional-51266689',
```

Comment:
Why is this url excluded?

Reply:
It's page-weight is too large so it fails the validation

Reply:
hmm, the other asset that is excluded is a bit unrealistic but this one isn't too bad in terms of realism, might be worth checking with @neildoughty / @gavinspence the number of valid to invalid assets just to check things aren't slowly getting worse.

We made the decision to exclude the other asset here: https://github.com/bbc/simorgh/issues/8104#issuecomment-756616723

Reply:
Please don't block the merge of this on the above 👍